app/models/form/lettings/pages/no_females_pregnant_household_lead_age_value_check.rb

```diff
@@ -16,4 +16,8 @@ def initialize(id, hsh, subsection)
   def questions
     @questions ||= [Form::Lettings::Questions::PregnancyValueCheck.new(nil, nil, self)]
   end
+
+  def affected_question_ids
```

Comment:
What does this represent? IMO the title of this method could be more descriptive.

Reply:
`ids_of_questions_triggering_validation` if this is still a question